envbuilder.go

```diff
@@ -89,7 +89,7 @@ func Run(ctx context.Context, options Options) error {
 	// Once the legacy environment variables are phased out, this can be
 	// reinstated to the previous default values.
 	if len(options.IgnorePaths) == 0 {
-		options.IgnorePaths = []string{"/var/run"}
+		options.IgnorePaths = []string{"/var/run", "/product_uuid", "/product_name"}
```

Comment:
Nit: would be good to have an explanation inline as well I think?

Reply:
Fixed!